droptools-example/src/main/java/com/bendb/example/resources/PostsResource.java

```diff
@@ -25,35 +26,61 @@
 @Path("posts")
 @Produces(MediaType.APPLICATION_JSON)
 public class PostsResource {
+
     @GET
-    public List<BlogPost> findPostsByTag(@QueryParam("tag") String tag, @Context DSLContext create) {
+    public List<BlogPost> findPostsByTag(@QueryParam("tag") String tag,
+                                         @QueryParam("all") boolean isAll,
```

Comment:
I didn't realize the method was named "findPostsByTag" - the "all" query param was meant to find all posts. I've cleared up the confusion by removing the query param entirely and reverting this method back to its original. 

I've instead modified the POST method to demonstrate the use of the alternate datasource instance.